libraries/SITL/SIM_VectorNav.h

```diff
@@ -0,0 +1,50 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  simulate serial VectorNav AHRS
+*/
```

Comment:
I try to keep a pattern of dropping in instructions on how to use the simulation in the top of these headers.  Examples include all of the simulated rangefinders (e.g. https://github.com/ardupilot/ardupilot/blob/master/libraries/SITL/SIM_RF_Wasp.h#L18) and RichenPower: (https://github.com/ardupilot/ardupilot/blob/master/libraries/SITL/SIM_RichenPower.h#L18)

The RichenPower one also includes how to run the relevant autotest.

I've found these instructions really help when you want to create a setup for testing.

Reply:
done